docs/en/faq.md

```diff
@@ -1,332 +1,169 @@
-# FAQs
+# Frequently Asked Questions
 
-## Why should I install a computing engine like Spark or Flink?
+## Do I need to install engines like Spark or Flink to use SeaTunnel?
+No, SeaTunnel supports Zeta, Spark, and Flink as options for the integration engine. You can choose one of them. The community especially recommends using Zeta, a new-generation high-performance engine specifically built for integration scenarios.
+The community provides the most support for Zeta, which also has richer features.
```

Comment:
The answer is not always no, it depends on the engine used by the user. If you use Zeta, you need to install the engine, just like Flink/Spark.